src/config.ts

```diff
@@ -11,6 +11,7 @@ export const config = {
   algoliaIndexName: process.env.REACT_APP_ALGOLIA_INDEX_NAME || 'product',
   compareKeys: process.env.REACT_APP_COMPARE_KEYS?.split(',') || [],
   endpointURL: process.env.REACT_APP_ENDPOINT_URL || 'api.moltin.com',
+  b2b: false,
```

Comment:
Please add the check for enabling this via an env var too so we can set it for deployments:

```suggestion
  b2b: process.env.REACT_APP_B2B_ENABLE || false,
```

Reply:
@yulia-dnistrian @yurii-khovzun I just noticed the config for the languages and currencies are not accepted as env vars. Any thoughts to how we can add those too?

If we manage to pull them from the APIs at build time though, we can eliminate this altogether